lib/pages/help_list/help_list_page.dart

```diff
@@ -63,6 +67,15 @@ class _HelpListPageState extends State<HelpListPage> {
     });
   }
 
+  _switchCards() {
+    setState(
```

Comment:
We should think in an alternative to avoid using setState and use Bloc to handle to state changes
Let's plan that in the next sprints.